test/e2e/tests/eg_upgrade.go

```diff
@@ -53,6 +53,7 @@ var EGUpgradeTest = suite.ConformanceTest{
 			depNS := "envoy-gateway-system"
 			lastVersionTag := os.Getenv("last_version_tag")
 			if lastVersionTag == "" {
+				// Use v1.0.2 instead of v1.1.2 due to https://github.com/envoyproxy/gateway/issues/4336
```

Comment:
Use v1.0.2 instead of v1.1.2 due to https://github.com/envoyproxy/gateway/issues/4336

https://github.com/envoyproxy/gateway/actions/runs/11048844503/job/30693628918?pr=4270
https://github.com/envoyproxy/gateway/actions/runs/11048844503/job/30693631030?pr=4270

Reply:
we should automatically this, can you create an issue for this?

Reply:
this is merged, can we move this to v1.1.2 ?

Reply:
> this is merged, can we move this to v1.1.2 ?

We can't because the upgrade test uses the published v1.1.2 version, not the latest version. We can move to v1.1.3 later after #4336 is merged into v1.1 branch and published with v1.1.3.